openebs/types/v1/types.go

```diff
@@ -16,7 +16,13 @@ limitations under the License.
 
 package v1
 
-//VolumeSpec holds the config for creating a VSM
+const (
+	PVCLabelsApplication = "volumeprovisioner.mapi.openebs.io/application"
+	PVCLabelsREplicaTopKeyDomain = "volumeprovisioner.mapi.openebs.io/replica-topology-key-domain"
```

Comment:
`PVCLabelsReplicaTopKeyDomain` instead of `PVCLabelsREplicaTopKeyDomain` and `PVCLabelsReplicaTopKeyType` instead of `PVCLabelsREplicaTopKeyType`

Reply:
Thank you!! fixed it